third_party/llvm/llvm.autogenerated.BUILD

```diff
@@ -263,6 +261,8 @@ llvm_target_list = [
             ("-gen-asm-matcher", "lib/Target/AMDGPU/AMDGPUGenAsmMatcher.inc"),
             ("-gen-disassembler", "lib/Target/AMDGPU/AMDGPUGenDisassemblerTables.inc"),
             ("-gen-pseudo-lowering", "lib/Target/AMDGPU/AMDGPUGenMCPseudoLowering.inc"),
+            ("-gen-searchable-tables", "lib/Target/AMDGPU/AMDGPUGenSearchableTables.inc"),
+            ("-gen-global-isel", "lib/Target/AMDGPU/AMDGPUGenGlobalISel.inc"),
```

Comment:
I'm not sure what is the right thing to do here, but I suspect that hand-modifying this file with "autogenerated" in the name is not going to work well.

@d0k, do you know the right thing to do here?

Reply:
The 2 lines are added so missing TableGen tables in AMDGPU backend is generated. I'd like to understand how this file is autogenerated as I failed identify relevant scripts in TensorFlow repo.

Reply:
The generator is not part of open source tensorflow (annoying, I know) so changing the generated file is the right approach.

@jlebar Ping me when this lands so I can fix the generator script, otherwise it'll get overwritten the next time someone runs it.

Reply:
Thanks.  Can we just eagerly fix the generator script?  It should generate the same thing as this PR anyway, so there should be no conflict upon rebasing/merging.

Reply:
I'll create an internal patch to fix the generator script.